cmd/recipe-checker/main.go

```diff
@@ -53,7 +55,12 @@ func main() {
 
 	router := mux.NewRouter()
 	router.Path("/").HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		data := getRecipeList()
+		ctx := req.Context()
+		userAuthToken, _ := headers.GetUserAuthToken(req)
```

Comment:
We don't care about this error. If there is an error then it will be `HEADER NOT FOUND` which is fine. We use whatever value is available and let the dataset API decide if we should have access or not.